### DISCUSSION TITLE
Can't find email address associated with username

### DISCUSSION THREAD

**Discussion by abhidotnet** [community/community#22676] [New to GitHub]
<p>I have been given a repository. I have also been given a username and password. Each time, I try to sign in the repository, I am met with the following page.</p>
<p>How can I find the email the code is being sent to?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d3b87ba36164e70411b188f2e64d6334fb021e60.png" data-download-href="/uploads/short-url/ucXZWSkxZvcbUbCO6asg90AhHEc.png?dl=1" title="deviceveri" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/164457/181072565-cfcef37b-ae87-4ca2-85be-8d17ae9cd515.png" alt="deviceveri" data-base62-sha1="ucXZWSkxZvcbUbCO6asg90AhHEc" width="486" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d3b87ba36164e70411b188f2e64d6334fb021e60_2_486x500.png, https://aws1.discourse-cdn.com/github/original/2X/d/d3b87ba36164e70411b188f2e64d6334fb021e60.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/d/d3b87ba36164e70411b188f2e64d6334fb021e60.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d3b87ba36164e70411b188f2e64d6334fb021e60_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">deviceveri</span><span class="informations">726×746 32.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/abhidotnet">@abhidotnet</a>, looks like the GitHub repository you are trying to access has 2FA enabled. You can asked the previous owner to disable it or change the notification method to one that <em>you</em>  have access to.</p>
<p>I hope this helps!</p>

**Comment by abhidotnet** [comment]
<p>Is there anyway, you could get me the <strong>previous owner’s email id</strong>? I could write to that email. This is a non-profit and I need to make sure that the code in the repository is the same that is running on the site. We have backlog that has not been checked into the repository and that information was not documented.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi! :wave:</p>
<p>This isn’t two factor authentication. This is a device verification message, and it requires that you have access to the email address on file. You will need to ask whomever it was who gave you the password to also give you access to the email address. The hint that is available there on the screen is as much information as we can provide about the email address.</p>
